src/app/dim-ui/SpecialtyModSlotIcon.tsx

```diff
@@ -21,19 +23,57 @@ function mapStateToProps() {
 }
 type Props = ProvidedProps & StoreProps;
 
-function SpecialtyModSlotIcon({ item, className, defs }: Props) {
+function SpecialtyModSlotIcon({ item, className, lowRes, defs }: Props) {
   const specialtySocket = getSpecialtySocket(item);
   const emptySlotHash = specialtySocket?.socketDefinition.singleInitialItemHash;
   const emptySlotIcon = emptySlotHash && defs.InventoryItem.get(emptySlotHash);
   return emptySlotIcon ? (
-    <div
-      className={`${className} ${styles.specialtyModIcon}`}
-      title={emptySlotIcon.itemTypeDisplayName}
-      style={bungieBackgroundStyle(
-        emptySlotIcon.displayProperties.icon,
-        'linear-gradient(#0005, #0005)' // forced dark background to help w/ visibility
-      )}
-    />
+    <PressTip elementType="span" tooltip={emptySlotIcon.itemTypeDisplayName}>
+      <div
+        className={`${className} ${styles.specialtyModIcon} ${lowRes ? styles.lowRes : ''}`}
```

Comment:
Not necessary but this can also be: 

```tsx
clsx(className, styles.specialtyModIcon, {[styles.lowRes]: lowRes})
```

Reply:
artifact of it being simpler at some point and saying "i don't need clsx" hhh~